BEP188.md

```diff
@@ -0,0 +1,52 @@
+# BEP188: Early Broadcast Mature block for in-turn validators.
+
+## 1. Summary 
+
+This BEP introduces a new block broadcast strategy for validators to improve the network capacity and stability.  
+
+## 2. Status
+
+Draft 
+
+## 3. Motivation
+
+Currently, a mined block will not be broadcasted until the timestamp header.Time is reached, even though the mining process is finished. If an in-turn validator spends too much time in the importing phase, it may not have enough time to mine its own block. In extreme cases, it could generate an empty block or even miss its block. This could be a challenge to the stability of the blockchain.With this BEP, validators could begin to import a block earlier and have more time to mine a new block.
+
+## 4. Specification
+
+### 4.1 Broadcast strategy 
+
+The current block production process is roughly shown in the figure below:
+
+![mining_process_normal](./assets/BEP-188/mining_process_normal.png)
+
+
+
+As we can see, even though a block was mined before header.Time, it will delay to be broadcasted after header.Time. Then the next in-turn validator starts importing and mining after it receives the block. Actually, a block mined by the in-turn validator can be broadcast as soon as the mining process is finished to let other validators start importing in advance.
+But if we early broadcast without limit, the block production process may be like the figure below in case each validator finished mining very quickly because they ran out of gas.
+
+![broadcast_withoutlimit](./assets/BEP-188/broadcast_withoutlimit.png)
+
+
+
+In this case, more than one block was mined in one block production cycle, which will break the current rule.Let’s see how it works if we limit the block broadcast time to after its parent’s header.Time under the same condition. 
+
+![early_broadcast_with_limit](./assets/BEP-188/early_broadcast_with_limit.png)
+
+Block N finished mining in advance because it ran out of gas and then broadcast without delay.Block N+1 also finished in advance and before blockN.header.Time, but it must delay broadcast to be after blockN.header.Time. In this way, the block production speed won’t be too quick to break the rule.
+In summary, when a block mined by an in-turn validator, iftime is after its parent block’s header.Time, broadcast it without delay.time is before its parent block’s header, broadcast it when the parent block’s head.Time is reached.
```

Comment:
`iftime`, typo

Reply:
`In summary, when a block mined by an in-turn validator, iftime is after its parent block’s header.Time, broadcast it without delay.time is before its parent block’s header, broadcast it when the parent block’s head.Time is reached.`
this sentence is not quite clear, can be replaced by:
`In summary, when a block is mined by an in-turn validator and the time is after its parent block’s header.Time, it can broadcast it immediately. But if the time is before its parent block’s header, it should not broadcast until the parent block’s head.Time is reached.